src/main/java/info/blockchain/wallet/shapeshift/data/Trade.java

```diff
@@ -39,19 +39,7 @@
     private Quote quote;
 
     public STATUS getStatus() {
-
-        // TODO: 11/10/2017 There must be an easier way to do this.
-        if (status.equals(STATUS.NO_DEPOSITS.toString())) {
-            return STATUS.NO_DEPOSITS;
-        } else if (status.equals(STATUS.RECEIVED.toString())) {
-            return STATUS.RECEIVED;
-        } else if (status.equals(STATUS.COMPLETE.toString())) {
-            return STATUS.COMPLETE;
-        } else if (status.equals(STATUS.RESOLVED.toString())) {
-            return STATUS.RESOLVED;
-        } else {
-            return STATUS.FAILED;
-        }
+        return STATUS.valueOf(status);
```

Comment:
Haha thanks!

Reply:
Actually I messed this up, hang on a second